django/contrib/admin/static/admin/js/actions.js

```diff
@@ -109,7 +109,7 @@
             lastChecked = target;
             updateCounter();
         });
-        $('form#changelist-form table#result_list tr').find('td:gt(0) :input').change(function() {
+        $('form#changelist-form table#result_list tr').on('change', 'td:nth-child(n+2) :input', function() {
```

Comment:
:nth-child(n+2) performs the same selection as :gt(0). I checked your question about removing the delete action and it works the same way (if there is no checkbox column, then 'actions' is disabled). But actually, I also checked the speed optimization, and using the former :gt(0) also improves the loading time in the same way, so there isn't an actual reason to change it. It could be left as :gt(0) and use the unique handler to speed up the code. That would leave the code more clear.

How do I modify a PR? Should I commit a new change? Or undo this one?

Thanks!